stories/ag/ToggleDropDown/ToggleDropDown.js

```diff
@@ -0,0 +1,62 @@
+import React from 'react'
+import { ToggleDropDown } from 'react-ui-ag/src'
+import StoryBookTheme from '../../theme/Storybook.css'
+
+const DefaultToggleDropDown = (
+  <div
+    className={StoryBookTheme['Story-center']}
+    style={
+      {
+        width: '100%',
+        height: '500px',
+        border: 'solid 1px black',
+        display: 'flex',
+        'align-items': 'flex-end',
```

Comment:
This property needs fixed
![screen shot 2018-01-22 at 8 16 25 am](https://user-images.githubusercontent.com/10748767/35231047-9b4b7adc-ff4c-11e7-8b24-b9be25167ae3.png)